addons/mail/static/src/models/thread/thread.js

```diff
@@ -819,6 +819,13 @@ function factory(dependencies) {
             this.unfollow();
         }
 
+        onClickOpen() {
```

Comment:
not a good name, open is the result not the element (commit: Move onClickOpen for thread)